whitepaper/chapter01.md

```diff
@@ -0,0 +1,140 @@
+
+<a name="1.5.3"></a>
+### 1.5.3 Principles for Cloud Native Telco Infrastructure and Applications
+
+At its core, the evolution from box-centric to process and function-centric requires an evaluation of the OSI Layers through a cloud native lens. The best general practices of cloud native infrastructure deployment and application design should also be applied to networking infrastructure Two key shifts required for CNFs to function well are deployment pipelines and the use of declarative programming and scripts in configuration. By pipelines, we mean the use of continuous integration (CI) and continuous delivery / deployment (CD) platforms. This is fundamental because it allows for rapid iteration, proper versioning, easier rollbacks, canary testing, and many other capabilities that are presently complex and often not possible in deploying VNFs. By declarative language, we mean a shift to intelligent orchestration that is better able to  maintain required service levels and maximise efficiencies based on available resources. This will ultimately mean a transition of network intelligence and design  - function, description, and configuration -  into other parts of Kubernetes (Helm Charts and CRDs).
+
+<a name="1.5.4"></a>
+### 1.5.4 Evolving the Stack From VNFs to CNFs
+
+The emergence of Kubernetes as the most widely used container orchestration layer allows telecommunications operators to shift their infrastructure over from more bespoke, expensive, and higher friction stacks to a de facto standard platform while also improving performance and resiliency. The malleability of Kubernetes should allow a gradual transition from legacy VNFs built on OpenStack or VMWare over to a Kubernetes telco stack that accommodates all aspects of their business - including legacy VNFs, which can be easily managed via an abstraction layer on top of Kubernetes (KubeVirt/Virtlet/Openstack).
+
+Back-office applications, such as BSS and OSS functions, can run directly on Kubernetes and will enjoy all the same benefits that enterprise applications derive from Kubernetes. Operators should also be able to run real-time, critical functions on Kubernetes. CNFs must be able to  run  on Kubernetes and benefit directly and strongly from cloud native principles. It is important to note that, in theory, Kubernetes and containerisation should allow for true cloud-blind immutability and portability. This will only be true if conforming Kubernetes versions and configurations are used and supported by operators. That said, creating a conformance model for Kubernetes for telecommunications is non-trivial, requiring validation and certification of key elements such as network software (CNI plug-in, network service mesh, etc.) and other aspects.
```

Comment:
"CNFs must be able to  run  on Kubernetes and benefit directly and strongly from cloud native principles."
I would remove the reference to k8s there since it is referred to in the previous sentence. Also, even if K8S is a flagship for CNCF, nothing prevents that in 2-3 years it might be replaced by a "shinier platform", cloud-native principles should still prevail.